screens/SelectCountryScreen.js

```diff
@@ -1,39 +1,54 @@
-import React from 'react';
+import React, { Component } from 'react';
 import {
     View,
     Text,
     StyleSheet,
     ImageBackground,
     Picker,
+    TouchableOpacity,
 } from 'react-native';
+import CountryProvider from "../provider/CountryProvider";
 
-export default function SelectCountryScreen() {
+export default class SelectCountryScreen extends Component {
+  state = { countries: [] }
+
+  componentDidMount() {
+    CountryProvider.getCountries()
+      .then(countries => this.setState({ countries }))
+      .catch((err) => console.log(err));
+  }
+
+  renderCountryItem(country) {
+    return (<Picker.Item key={country.id} label={country.name} value={country.id} />);
+  }
+
+  
+  render(){
     return(
       <ImageBackground source={require("../img/bg-img.jpg")} style={styles.bgImg}>
         <Text style={styles.title}>TopRating</Text>
-        <Text style={styles.introText}>Search for differents countrys and get to know their top rating musics!</Text>
+        <Text style={styles.introText}>Search for differents countries and get to know their top rating musics!</Text>
         <View style={styles.separator}></View>
         <View style={styles.container}>
           <Text style={styles.textStart}>Get Started:</Text>
           <Picker style={styles.picker} mode={"dropdown"}>
-            <Picker.Item label="XX" value="none" />
-            <Picker.Item label="Brazil" value="br" />
-            <Picker.Item label="Italy" value="it" />
-            <Picker.Item label="United States" value="us" />
-            <Picker.Item label="United Kingdom" value="uk" />
-            <Picker.Item label="World Wide" value="xw" />
+            {this.state.countries.map((country) => this.renderCountryItem(country))}
           </Picker>
+          <TouchableOpacity style={styles.searchButton} onPress={() => {}}>
```

Comment:
Deixar um comentário com TODO onde existir pendência. Nesse caso, deixar um `/* TODO */` dentro do comando do onPress, pois aí sabemos que você mandou uma modificação que existe uma pendência.
```suggestion
          <TouchableOpacity style={styles.searchButton} onPress={() => { /* TODO implement */ }}>
```